lib/pbench/test/unit/server/test_file_tree.py

```diff
@@ -328,8 +332,11 @@ def mock_rmtree(path: Path, ignore_errors=False):
             assert ignore_errors
             assert path == incoming / f"{tarball_name}.unpack"
 
-        def mock_run(self, command, dir_path, exception, dir_p):
-            raise subprocess.TimeoutExpired("tar", 43)
+        @staticmethod
+        def mock_run(command, dir_path, exception, dir_p):
+            verb = "tar"
+            assert command.startswith("tar")
+            raise subprocess.TimeoutExpired(verb, 43)
```

Comment:
You should reference `verb` instead of `"tar"` at line 338 (like you do at line 339).